pkg/devspace/pipeline/pipeline.go

```diff
@@ -224,10 +224,13 @@ func (p *pipeline) StartNewDependencies(ctx devspacecontext.Context, dependencie
 
 	// Start sequentially
 	if options.Sequential {
+		ctx.Log().Info("Deploying dependencies sequentially")
```

Comment:
Can we switch both log statements to `Debug()`/`Debugf()`?

Reply:
Okay.

Reply:
Done